test/integration/performance_test.rb

```diff
@@ -24,7 +24,7 @@
       Sidekiq::Cron::Job.create(args.merge(name: "Test#{i}"))
     end
 
-    @poller = Sidekiq::Cron::Poller.new
+    @poller = Sidekiq::Cron::Poller.new({})
```

Comment:
And then, once its optional, we could just init the poller using the old way.

```suggestion
    @poller = Sidekiq::Cron::Poller.new
```